twitter_post_block.py

```diff
@@ -69,9 +63,12 @@ def _post_tweet(self, payload):
 
         status = response.status_code
         if status != 200:
+            response = response.json()
             self._logger.error(
```

Comment:
I've been thinking that we log to error to freely. Maybe these should be warnings?